### ISSUE TITLE
PC DOS 2000 power.exe causes stream of errors

### ISSUE THREAD

**Issue by rderooy** [joncampbell123/dosbox-x#1503]
**Describe the bug**
By default, when the PC DOS 2000 setup.exe detects a APM bios, it adds the following line to CONFIG.SYS:

```DEVICE=C:\DOS\POWER.EXE```

When you then boot the disk image, you get a continuous rapid stream of errors in the log:
```LOG: APM BIOS warning: CPU IDLE called with IF=0, not HLTing```

Tried setting the various apm* config options, but other then "apmbios=false" or "apmbios allow realmode=false", nothing made any difference.

**To Reproduce**
* Install PC DOS 2000 in a disk image, with the APMBIOS config option set to default.

**Expected behavior**
No errors

**Environment (please complete the following information):**
- Fedora 31
- DOSBox-X 0.83.1-git (todays build)

**Additional context**
Part of the log:

```
LOG: APM BIOS: Connected to real-mode interface
LOG:  349935590 ERROR BIOS:INT16:Unhandled call 6F
LOG: INT 15h mouse initialized to 3-byte protocol
LOG:  350070154 ERROR INT10:unknown RIL port selection 720
LOG:  350398236 ERROR INT10:Function CC00 not supported
LOG: INT 15h mouse initialized to 3-byte protocol
LOG: INT 15h reporting mouse device ID 0x00
LOG: APM BIOS warning: CPU IDLE called with IF=0, not HLTing
LOG: APM BIOS warning: CPU IDLE called with IF=0, not HLTing
LOG: APM BIOS warning: CPU IDLE called with IF=0, not HLTing
LOG: APM BIOS warning: CPU IDLE called with IF=0, not HLTing
LOG: APM BIOS warning: CPU IDLE called with IF=0, not HLTing
```

**Comment by Wengier** [comment]
@rderoroy These are basically warnings in the log file related to APM BIOS, not real errors shown to the user. Have you tested if POWER.EXE works as expected?

**Comment by rderooy** [comment]
@Wengier I'm not sure how I can test if POWER.EXE works as expected. It is not like DOSBox has a power meter :-)

I did try to look at CPU utilisation on my host system, but a idle DOSBox-X running PC-DOS 2000 uses around 7% CPU regardless of which power mode I specify with POWER.EXE (Min, Avg, Max, Off).

What is noticeable is the messages in the log, there are tons of messages being logged every second. pressing enter in the terminal window from where I started DOSBox-X, and where the LOG messages are being written does not even show me an empty line as it normally would, it is writing that fast.

**Comment by Wengier** [comment]
@rderooy I tested POWER.EXE myself in Windows 10. So if POWER.EXE is installed in CONFIG.SYS, and boot the PC DOS 2000 disk image in DOSBox-X:

With "POWER ADV": DOSBox-X will use around 30% CPU;
With "POWER STD": DOSBox-X will use less than 10% CPU.

I tried to switch between the two modes (ADV and STD) from the command line and monitor the CPU real-time usage of DOSBox-X, and I found that this behavior is very consistent. So my conclusion is that POWER.EXE is indeed working in DOSBox-X, at least in Windows 10.

**Comment by rderooy** [comment]
@Wengier thanks for trying that. So I guess the only thing to do in that case is to reduce the logging. perhaps change the message to Debug level. I'm sure you must have seen in your console how much logging it is generating.

**Comment by Wengier** [comment]
@rderooy I can see that in the log of course, but I do not know if any pull request is desired at this time though, because I think @joncampbell123 is currently still preparing for the release.

**Comment by rderooy** [comment]
@Wengier Sure, I understand. Thanks again for having a look.

**Comment by joncampbell123** [comment]
Go ahead and submit the pull request and I'll pull when the release is up

**Comment by Wengier** [comment]
I have noticed that the new release is now up in the release page, which is great! Thanks for your efforts!

About the pull request, I intended to change only one statement in the file src/ints/bios.cpp, but Github says in the header "We’ve detected the file has mixed line endings. When you commit changes we will normalize them to Windows-style (CRLF)." Basically it will automatically change all line endings of this file. I never saw this message in my previous pull requests, and there does not seem to be any option to change this behavior. Interesting.
